src/hardware/addon-protocols/can.md

```diff
@@ -61,6 +63,14 @@ Each piece of information related to the third-party device must be sent individ
 
 Note: See [Appendix A](#appendix-a-raw-message-data-example-for-iox-can) for an example of raw message data.
 
+#### Handshake
+An initial Handshake **is required** in order for the GO device to accept MyGeotab API calls to produce CAN messages from the IOX-CAN. If this functionality is not required, the handshake can be skipped. Ignition must be on for the handshake process.
```

Comment:
"If this functionality is not required, the handshake can be skipped." I would prefer we don't say this. The identification and handshake is part of the protocol. The fact that we allow status data to be saved without this needing to be completed is something we may wish to change/enforce at some point in time. Being able to identify the external device is always helpful from a data mining perspective.

Reply:
Probably better to remove the "No initial handshake is required to communicate third-party data to the IOX-CAN" in the other section as well.